src/ddbToEs/index.ts

```diff
@@ -13,9 +14,9 @@ const REMOVE = 'REMOVE';
 // This lambda picks up changes from DDB by way of DDB stream, and sends those changes to ElasticSearch Service for indexing.
 // This allows the FHIR API Server to query ElasticSearch service for search requests
 
-export async function handleDdbToEsEvent(event: any) {
-    const ddbToEsHelper = new DdbToEsHelper();
+export async function handleDdbToEsEvent(event: any, ElasticSearch: Client) {
```

Comment:
I think that this function shouldn't take the client as a param. It's awkward to pass the client to every single invocation. 

It can be changed to a class that takes the `esClient` as a param and keeps a reference to it. Then we can keep the original fn signature`handleDdbToEsEvent(event: any) => {...}`

That's similar to how other fwoa components work, with the difference that they don't take aws clients as params but they still init clients only once in constructors.